.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+This PR closes #{issue_number}.
```

Comment:
Too long. Closes #{issue_number} is enough.

Reply:
I like sentences that are grammatically correct :)

Reply:
Most importantly, does it actually work? After reading https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword it is unclear if anything can be before the keyword